docs/installation/SUSE.md

```diff
@@ -28,13 +28,17 @@ Execute one of the following commands to add the Virtualization repository:
 
 No extra repository is required for openSUSE 13.2 and later.
 
+Proceed to the "Installation" section.
+
 # SUSE Linux Enterprise
 
 Docker is available in **SUSE Linux Enterprise 12 and later**. Please note that
 due to its current limitations Docker is able to run only on **64 bit**
 architecture.
 
-## Installation
+Proceed to the "Installation" section.
+
+# Installation
```

Comment:
Can you change this back to a heading 2? This keeps the structure consistent with the other docs

Ideally, I think we should use the same structure as is used on the Ubuntu page; https://docs.docker.com/installation/ubuntulinux/, i.e. a "Prerequisites" section, with "openSUSE" and "SUSE Linux Enterprise" below it, but that's probably out of scope for this PR :smile: